CHANGELOG.md

```diff
@@ -11,6 +11,7 @@ This file is used to list changes made in each version of the AWS ParallelCluste
 - Track head node memory and root volume disk utilization using the `mem_used_percent` and `disk_used_percent` metrics collected through the CloudWatch Agent.
 - Add log rotation support for ParallelCluster managed logs.
 - Add support for customizing the cluster Slurm configuration via the ParallelCluster configuration YAML file.
+- Add health check manager and GPU health check, that can be activated through cluster configuration.
```

Comment:
We need to keep aligned this section and https://github.com/aws/aws-parallelcluster/blob/develop/CHANGELOG.md

Reply:
done in https://github.com/aws/aws-parallelcluster/pull/5177